doc/source/installation/yum_repo.rst

```diff
@@ -51,24 +51,30 @@ Installing |Percona Server| from Percona ``yum`` repository
 
    .. code-block:: bash
 
-     yum install http://www.percona.com/downloads/percona-release/redhat/0.1-6/percona-release-0.1-6.noarch.rpm
+      $ yum install http://www.percona.com/downloads/percona-release/redhat/0.1-6/percona-release-0.1-6.noarch.rpm
 
-   You should see some output such as the following: 
+   .. admonition:: Output example
+
+      .. code-block:: guess
+
+	 Retrieving http://www.percona.com/downloads/percona-release/redhat/0.1-6/percona-release-0.1-6.noarch.rpm
+	 Preparing...                ########################################### [100%]
+         1:percona-release        ########################################### [100%]
+
+   To install |Percona Server| with SELinux policies, install the :program:`Percona-Server-selinux-*.noarch.rpm` package instead:
 
    .. code-block:: bash
 
-     Retrieving http://www.percona.com/downloads/percona-release/redhat/0.1-6/percona-release-0.1-6.noarch.rpm
-     Preparing...                ########################################### [100%]
-        1:percona-release        ########################################### [100%]
+      $ yum install http://repo.percona.com/centos/7/RPMS/x86_64/Percona-Server-selinux-56-5.6.42-rel84.2.el7.noarch.rpm
 
-.. note:: 
+   .. note:: 
 
-  *RHEL*/*Centos* 5 doesn't support installing the packages directly from the remote location so you'll need to download the package first and install it manually with :program:`rpm`:
+      *RHEL*/*Centos* 5 doesn't support installing the packages directly from the remote location so you'll need to download the package first and install it manually with :program:`rpm`:
```

Comment:
This can be removed as CentOS 5 isn't supported anymore.

Reply:
Note removed